docs/migrating-from-redux-gtm.md

```diff
@@ -63,8 +65,8 @@ const metaReducer = createMetaReducer(eventsMap);
 
 ##### Now
 ```js
-const middleware = createMiddleware(eventsMap, GoogleTagManager);
-const metaReducer = createMiddleware(eventsMap, GoogleTagManager);
+const middleware = createMiddleware(eventsMap, GoogleTagManager());
```

Comment:
GoogleTagManager allows you to use another name for the dataLayer, we wanted to accommodate that. So now if you're using a name other than dataLayer you can specify it as a param.